catalogue_graph/tests/test_ingestor_deletions.py

```diff
@@ -1,67 +1,112 @@
 import polars as pl
+import pydantic
 import pytest
 
-from graph_remover import IDS_LOG_SCHEMA
 from ingestor.steps.ingestor_deletions import lambda_handler
 from tests.mocks import MockElasticsearchClient, MockSmartOpen, mock_es_secrets
 
 REMOVER_S3_PREFIX = "s3://wellcomecollection-catalogue-graph/graph_remover_incremental"
 
-
-def get_mock_remover_uri(pipeline_date: str) -> str:
-    return f"{REMOVER_S3_PREFIX}/{pipeline_date}/deleted_ids/catalogue_concepts__nodes.parquet"
-
-
 MOCK_EVENT = {
     "ingestor_type": "concepts",
     "pipeline_date": "dev",
     "index_date": "dev",
     "job_id": "dev",
 }
 
+MOCK_TIME_WINDOW_EVENT = {
+    **MOCK_EVENT,
+    "window": {"start_time": "2025-10-22T08:00", "end_time": "2025-10-22T08:15"},
+}
+
 
 def index_concepts(ids: list[str], index_name: str = "concepts-indexed-dev") -> None:
     for _id in ids:
         MockElasticsearchClient.index(index_name, _id, {})
 
 
-def get_indexed_concepts(index_name: str = "concepts-indexed-dev") -> dict:
-    return MockElasticsearchClient.indexed_documents[index_name] or {}
+def get_indexed_concept_ids(index_name: str = "concepts-indexed-dev") -> list[str]:
+    docs = MockElasticsearchClient.indexed_documents[index_name] or {}
+    return list(docs.keys())
 
 
 def mock_deleted_ids_log_file(pipeline_date: str) -> None:
-    mock_data = {
-        "timestamp": ["2025-04-03", "2025-04-03", "2025-04-07", "2025-04-07"],
-        "id": ["u6jve2vb", "amzfbrbz", "q5a7uqkz", "s8f6cxcf"],
-    }
-    df = pl.DataFrame(mock_data, schema=IDS_LOG_SCHEMA)
-    MockSmartOpen.mock_s3_parquet_file(get_mock_remover_uri(pipeline_date), df)
+    mock_data = ["u6jve2vb", "amzfbrbz", "q5a7uqkz", "s8f6cxcf"]
```

Comment:
Can we perhaps hand the mock data in, so in the test it's clear which ids are being mocked for deletion?